src/components/Team/TeamMemberListItem.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { User } from "../../types";
+import { ListItem, ListItemText } from "@material-ui/core";
+
+interface TeamMemberListItemProps {
+  member: User;
+}
+const TeamMemberListItem = ({ member }: TeamMemberListItemProps) => {
+  return (
+    <ListItem button key={`team_member_list_item_inner_${member.id}`}>
```

Comment:
I think there's no need to use a key here since you are no iterating and you are already providing this one key={`team_member_list_item_${member.id}`} for the outer element.

Reply:
Done.